formver.compiler-plugin/core/src/org/jetbrains/kotlin/formver/core/conversion/ProgramConverter.kt

```diff
@@ -120,9 +125,23 @@ class ProgramConverter(
 
         // Note: it is important that `body` is only set after `embedUserFunction` is complete, as we need to
         // place the embedding in the map before processing the body.
-        embedUserFunction(declaration.symbol, signature).apply {
-            body = stmtCtx.convertMethodWithBody(declaration, signature, returnTarget)
+        if (declaration.symbol.isPure(session)) {
+            embedPureUserFunction(declaration.symbol, signature).apply {
+                body = PureFunctionBodyEmbedding(Exp.IntLit(42), returnTarget)
```

Comment:
Please leave a TODO here to make it clear that this is a placeholder.  (Preferably, also make a variable that's called `placeholder` so that the extent of the placeholder is clear.)